content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -362,6 +362,41 @@ For more information on inter-pod affinity/anti-affinity, see the
 You may want to check [Taints](/docs/concepts/configuration/taint-and-toleration/)
 as well, which allow a *node* to *repel* a set of pods.
 
+## nodeName
+
+`nodeName` is the simplest form of node selection constraint, but due
+to its limitations its use is _not_ recommended.  `nodeName` is a
```

Comment:
this might be a bit strong. it would probably be better to say that it is not typically done, and to enumerate why, as you have

Reply:
I've changed this to "but due to its limitations it is typically not used."